### DISCUSSION TITLE
Why is recoil experimental?

### DISCUSSION THREAD

**Discussion by roccoluke** [facebookexperimental/Recoil#1544] [General]
Just wondering if there is a specific reason why recoil hasn't made it to beta or a further stage than experimental?

Compared to jotai which implements a similar atomic api, recoil has ~3x more weekly downloads on npm so there's definitely a preference here and reason to support this library further. Is recoil waiting for react 18 (maybe concurrent rendering) to evolve further and become more commonly used? Should we expect recoil to change drastically in upcoming versions or the vision is to remain using an api similar to the latest version?

Would ❤️ to hear more about this 🙏 

Thanks!

**Comment by Cpt-Falcon** [comment]
Well on a superficial level you can see all sorts of recoil features labelled "UNSTABLE", so I think they're still testing and refining various things. Also they haven't provided a roadmap to the community here so we have no idea what version 1.0 will contain or when that will occur.

The real problem is that its difficult to justify using an experimental code base in an important production project despite the fact that recoil is amazing. If I have a critical application that needs to work without fail, relying on an unstable/experimental code base is a hard sell compared to redux which has been hammered on and tested for years.

I think this really needs to get out of experimental ASAP so that we can feel comfortable using this technology in a production app without worrying about reliability.  I hope the recoil team comes out with both a roadmap and an estimate of when we're moving out of experimental.

**Reply by roccoluke** [reply]
I agree with justifying the wait time and that is the reason for asking. Also my concern is introducing breaking changes in the api that might be time consuming to continuously update.

**Reply by roccoluke** [reply]
> I agree with justifying the wait time and that is the reason for asking. Also my concern is introducing breaking changes in the api that might be time consuming to continuously update.
> 
> 

I understand that it takes time. But some information from the team would be great (blog updates, milestones on github or a roadmap).

**Comment by drarmstr** [comment]
Please see discussion in #1495
